docs/spec.openapi.yml

```diff
@@ -0,0 +1,658 @@
+                $ref: '#/components/schemas/ResStatus'
+
+  /file_delete:
+    post:
+      tags: ["File"]
+      description: Delete one or multiple files from the user account.
+      requestBody:
+        content:
+          Object:
+            schema:
+              $ref: '#/components/schemas/ReqFileDelete'
+      responses:
+        "200":
+          description: The file was deleted.
+          content:
+            object:
+              schema:
+                $ref: "#/components/schemas/ResStatus"
+        "500":
+          description: Internal server error
+          content:
+            object:
+              schema:
+                $ref: "#/components/schemas/ResStatus"
+        'default':
+          description: An error occurred. Returns error code and message. See possible responses at [Metadata API Spec endpoint `DELETE /files/{user_uuid}/{file_uuid}`](https://github.com/hawks-atlanta/metadata-scala/blob/main/docs/spec.openapi.yaml)
+          content:
+            Object:
+              schema:
+                $ref: '#/components/schemas/ResStatus'
+                
+  /file_move:
+    put:
+      tags: ["File"]
+      description: Move or rename the given file. If `targetDirectoryUUID` is provided the file will be moved. If `newName` is provided the name will be renamed. 
+      requestBody:
+        content:
+          Object:
+            schema:
+              $ref: '#/components/schemas/ReqFileMove'
+      responses:
+        "200":
+          description: The file was moved / renamed.
+          content:
+            object:
+              schema:
+                $ref: "#/components/schemas/ResStatus"
+        'default':
+          description: An error occurred. Returns error code and message. See possible responses at [Metadata API Spec endpoint `PUT /files/rename/{user_uuid}/{file_uuid}` and `PUT /files/move/{user_uuid}/{file_uuid}`](https://github.com/hawks-atlanta/metadata-scala/blob/main/docs/spec.openapi.yaml)
+          content:
+            Object:
+              schema:
+                $ref: '#/components/schemas/ResStatus'
+
+  # share
+
+  /share_file:
+    post:
+      tags:
+        - Share
+      description: Share the given file with the given user
+      requestBody:
+        content:
+          object:
+            schema:
+              $ref: "#/components/schemas/ReqShareFile"
+      responses:
+        "200":
+          description: The file was shared
+          content:
+            object:
+              schema:
+                $ref: "#/components/schemas/ResStatus"
+        'default':
+          description: An error occurred. Returns error code and message. See possible responses at [Metadata API Spec endpoint `POST /files/share/{user_uuid}/{file_uuid}`](https://github.com/hawks-atlanta/metadata-scala/blob/main/docs/spec.openapi.yaml)
+          content:
+            Object:
+              schema:
+                $ref: '#/components/schemas/ResStatus'
+
+  /share_remove:
```

Comment:
Maybe`unshare_file` its more clear

Reply:
You're right, 'just trying to make them all start with `share` 😅️